.storybook/test-runner.ts

```diff
@@ -0,0 +1,33 @@
+import type { TestRunnerConfig } from '@storybook/test-runner';
+import { injectAxe, checkA11y, configureAxe } from 'axe-playwright';
+
+/*
+ * See https://storybook.js.org/docs/writing-tests/test-runner#test-hook-api
+ * to learn more about the test-runner hooks API.
+ */
+const config: TestRunnerConfig = {
+  async preVisit(page) {
+    await injectAxe(page);
+  },
+  async postVisit(page) {
+    await configureAxe(page, {
+      // Skip some rules for now as we work on addressing them
+      rules: [
+        {
+          // TODO(#7108, #a11y-162) - turn this back on
+          id: 'color-contrast',
+          enabled: false,
+        },
+        {
+          // TODO(#a11y-1)
```

Comment:
What is this referencing?

Reply:
Ah, sorry, it is referencing https://github.com/open-path/hmis-accessibility/issues/1. I will add the link in the code.

Reply:
ah okay, thanks!